src/integrationTest/java/com/mongodb/hibernate/id/MongoIdFieldNameIntegrationTests.java

```diff
@@ -34,7 +37,7 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-@SessionFactory(exportSchema = false)
+@SessionFactory(exportSchema = false, useCollectingStatementInspector = true)
```

Comment:
Why is this needed?  I'm not clear what purpose it's serving here.

Reply:
Sorry. It is a new leftover today when I experimented using it to go about verification but I found it only collects MQL containing parameters, so its usage would not be compelling to verify. Deleted.